recipes/MADL-CPU-IntelMPI/docker/Dockerfile

```diff
@@ -0,0 +1,59 @@
+#Dockerfile for MDL (Microsoft Distributed Learners)
+
+FROM ubuntu:16.04
+MAINTAINER Saeed Maleki Todd Mytkowicz Madan Musuvathi Dany rouhana <https://github.com/Azure/batch-shipyard>
```

Comment:
Use your landing page url instead.

Reply:
ok.  I changed it to this for now until we have the landing page ready: <SymSGD project landing page>